pubcrawler/article.py

```diff
@@ -27,7 +27,7 @@ def get_text_from_tags(self, tag_name):
         if tags is not None:
             tags_text = []
             for tag in tags:
-                tags_text.append(tag.get_text("\n\n"))
```

Comment:
I don't know why a double linebreak was used here before. It was causing some formatting issues on abstracts that use nested elements for things like italics so I removed it.